selfdrive/test/helpers.py

```diff
@@ -82,7 +82,20 @@ def read_segment_list(segment_list_path):
 
 # Utilities for sanitizing routes of only essential data for testing car ports and doing validation.
 
+def sanitize_vin(vin: str): # remove sentitive data from VIN
+  VIN_SENSITIVE = 6 # (last 6 digits of vin are serial number https://en.wikipedia.org/wiki/Vehicle_identification_number)
```

Comment:
```suggestion
def sanitize_vin(vin: str):  # remove sentitive data from VIN
  VIN_SENSITIVE = 6  # (last 6 digits of vin are serial number https://en.wikipedia.org/wiki/Vehicle_identification_number)
```

for readability/PEP: https://peps.python.org/pep-0008/#inline-comments

also probably doesn't need to be inline at all

Reply:
any ruff rules for these?

Reply:
Not yet